frontend/src/v5/ui/routes/viewer/tickets/tickets.component.tsx

```diff
@@ -17,54 +17,78 @@
-			Etiam nec tempor nibh.
-			Nulla tempus turpis id libero malesuada, non sollicitudin nunc laoreet. In in fringilla
-			urna. Ut a felis a lorem bibendum tempus vel nec orci. Vivamus ut dolor ut diam interdum
-			fermentum. Cras gravida placerat ex eget faucibus. Suspendisse potenti. Proin tincidunt
-			aliquam faucibus. Nunc sit amet sapien urna.
-			Fusce blandit eleifend urna. Aliquam blandit, metus sed porta posuere, turpis velit mollis
-			quam, ac sagittis lacus erat et ex. Donec sollicitudin sapien est, sed rutrum nibh venenatis
-			nec. Duis et rutrum ipsum. Quisque eget lorem ornare, blandit ex sit amet, pulvinar velit.
-			Integer ipsum tellus, tincidunt sed leo at, iaculis pretium purus. Etiam a molestie dui.
-			Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam venenatis quis mi non efficitur.
-			Donec malesuada maximus sapien ut fermentum. Nulla porttitor aliquet porttitor. Quisque interdum
-			mauris felis, sit amet luctus mi ultrices id. Maecenas rhoncus magna nec massa auctor commodo.
-			Maecenas posuere elit quis nisl iaculis fermentum. Proin aliquet quam et blandit sollicitudin.
-			In sit amet hendrerit ipsum.
-		</CardContent>
-	</CardContainer>
-);
+export const Tickets = () => {
+	const { teamspace, project, containerOrFederation } = useParams<ViewerParams>();
+	const isFederation = FederationsHooksSelectors.selectContainersByFederationId(containerOrFederation)?.length === 1;
+	const tickets = isFederation
+		? FederationsHooksSelectors.selectFederationTickets(containerOrFederation)
+		: ContainersHooksSelectors.selectContainerTickets(containerOrFederation);
+
+	useEffect(() => {
+		if (isFederation) {
+			FederationsActionsDispatchers.fetchFederationTickets(teamspace, project, containerOrFederation);
+		} else {
+			ContainersActionsDispatchers.fetchContainerTickets(teamspace, project, containerOrFederation);
+		}
+	}, [containerOrFederation]);
+
+	const [selectedTypes, setSelectedTypes] = useState<string[]>([]);
+
+	const toggleSelectedType = (type: string) => {
+		if (selectedTypes.includes(type)) {
+			setSelectedTypes(selectedTypes.filter((selectedType) => selectedType !== type));
+		} else {
+			setSelectedTypes(selectedTypes.concat(type));
+		}
+	};
+
+	const getFilteredTickets = () => {
+		if (selectedTypes.length === 0) return tickets;
+		return tickets.filter(({ type }) => selectedTypes.includes(type));
+	};
+
+	const getTypesByCount = () => countBy(tickets.map(({ type }) => type));
+
+	return (
+		<CardContainer>
+			<CardHeader>
+				<TicketsIcon fontSize="small" />
+				<FormattedMessage id="viewer.cards.tickets.title" defaultMessage="Tickets" />
+			</CardHeader>
+			<CardContent autoHeightMax="100%">
+				{!tickets?.length ? (
+					<EmptyCardMessage>
+						<FormattedMessage id="viewer.cards.tickets.emptyList" defaultMessage="No entries have been created yet" />
+					</EmptyCardMessage>
+				) : (
+					<>
+						<SearchValues>
```

Comment:
not sure if '<searchValues />' and '<searchvalue />' are the most appropiate name casue we are sure we are adding more searching capabilities, these might make the code a bit ambiguous.